tests/converter/test_converter.py

```diff
@@ -0,0 +1,38 @@
+import os
+import shutil
+import importlib
+
+from mindmeld.components import NaturalLanguageProcessor
+from mindmeld.test import TestConversation
+from mindmeld.converter.dialogflow import DialogflowConverter
+
+
+def test_df_converter():
+    df_project_path = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "dialogflow_sample_project"
+    )
+
+    # This is the dialogflow app converted to mindmeld app
+    mm_df_path = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "mm_df_converted_project"
+    )
+
+    df_init = DialogflowConverter(df_project_path, mm_df_path)
+    df_init.convert_project()
+
+    mm_df_nlp = NaturalLanguageProcessor(app_path=mm_df_path)
+    mm_df_nlp.build()
+    mm_df_nlp.dump()
+
+    mm_df_app = importlib.import_module('mm_df_converted_project').app
+    mm_df_app.lazy_init(mm_df_nlp)
+
+    conv = TestConversation(app=mm_df_app)
+    conv.process("what is my balance")
+    conv.assert_text("Here's your latest balance:")
+    conv.assert_domain("app_specific")
+    conv.assert_intent("accountbalancecheck_en")
+    conv.assert_frame({})
+
```

Comment:
we should also check in the nlp objects that:
- all domains, intents, entities are converted correctly
- the default domain is converted correctly

Reply:
to clarify, for #3 we don't need to check all responses, but add a few more to make sure that the edge cases are addressed.

Reply:
done with the first three points, point 4 is not supported in this PR